server/src/main/java/org/elasticsearch/action/admin/indices/rollover/TransportRolloverAction.java

```diff
@@ -123,7 +123,8 @@ protected void masterOperation(
             .clear()
             .indicesOptions(IndicesOptions.fromOptions(true, false, true, true))
             .docs(true)
-            .indexing(true);
+            .indexing(true)
+            .store(true);
```

Comment:
Do we need this? Can we instead use `shard.docs.getTotalSizeInBytes()` like is done in `buildStats`? That is according to comments more reliable.

Reply:
I thought the same, but since we already use the store stats for the ClusterInfo I thought it made sense to use the same source of information. I'll change it.